### PR TITLE
ASGS 2016 vector tile regionmapping layers

### PR DESCRIPTION
**Pull request by steve9164**
Here's a preview site with all of the layers (in ASGS 2016): http://nationalmap-regionmapping-test.terria.io/

~~Builds are failing due to reference to a git URL of nationalmap-catalog.~~

Note: This is ready for review, but not for merging.
Before merging:
- [ ] Review changes
- [ ] Fix logging on server
- [ ] Change production vector tile server alias
- [ ] Update URLs in `regionMapping.json`

### CODE REVIEW

File: wwwroot/data/regionMapping.json [hunk 1]
```diff
@@ -51,14 +48,60 @@
             ],
             "nameProp": "SA2_NAME11"
         },
+        "SA1_2016": {
```
Comment (kring) [inline wwwroot/data/regionMapping.json]: Should this be `SA1` and the current SA1 renamed to `SA1_2011`?  I'm not sure if it matters much, but it seems like the latest data should have the best name.

**Comment by kring** [comment]
This looks good to me.  @tobybellwood may want to take a look.

**Comment by tobybellwood** [comment]
Thanks @kring - we can't path sa1/sa2 etc yearless shortcodes to the 2016 versions until after the data comes out.  We can have them there as their explicit versions, but any shortcodes can't switch until the June 2017 Census release (as the first real data release on the boundaries)

**Comment by steve9164** [comment]
After a quick clarification over Slack with @tobybellwood:
- At the moment, don't update aliases such as `sa1` or `sa3_code` to use 2016 geometries as there is no data that corresponds to 2016 geometries yet (until Census is released)
- At the point that Census data is released, these aliases should be updated

**Comment by steve9164** [comment]
Slackbot will help me revet that commit ready for 2016 Census release:
image

**Comment by ajbar7** [comment]
Can we all ensure that we roll out 2016 LGAs?

http://www.abs.gov.au/AUSSTATS/abs@.nsf/DetailsPage/1270.0.55.003July%202016?OpenDocument

**Pull request closed without merging**